src/main/java/bio/terra/service/snapshotbuilder/SnapshotRequestDao.java

```diff
@@ -47,7 +46,6 @@ public class SnapshotRequestDao {
       (rs, rowNum) ->
           new SnapshotAccessRequestResponse()
               .id(rs.getObject(ID, UUID.class))
-              .datasetId(rs.getObject(DATASET_ID, UUID.class))
```

Comment:
Could you also remove or fix the references to "dataset id" on lines 93 and 96 below?